app/src/test/scala/org/alephium/explorer/api/model/ApiModelSpec.scala

```diff
@@ -48,6 +47,23 @@ class ApiModelSpec() extends AlephiumSpec with Generators {
     }
   }
 
+  it should "ConfirmedTransaction" in {
+    forAll(transactionGen) { tx =>
+      val expected = s"""
+       |{
+       |  "type": "confirmed",
```

Comment:
Let's use capitalized word for types, which is the case for most languages. In JavaScript, classes are named in `Foo` as far as I see